### PR TITLE
Ethan/i18n spanish

### PR DESCRIPTION
**Pull request by Ethanol48**

### CODE REVIEW

**Comment by BenraouaneSoufiane** [comment]
Too late

**Comment by BenraouaneSoufiane** [comment]
@daimo-eth why you delete my comment?
I'm warning you from using my code without paying me (400 USDC), I'm not working here for free

**Comment by BenraouaneSoufiane** [comment]
@Ethanol48 pay attention to copy past my submission as your proper work

**Comment by BenraouaneSoufiane** [comment]
I've asked for my rights with professional way, you ignored my, my submission, my work hard at all

**Comment by Ethanol48** [comment]
I haven't used your code at all.
You don't have even a pull request

**Comment by Ethanol48** [comment]
> @daimo-eth why you delete my comment?
> 
> I'm warning you from using my code without paying me (400 USDC), I'm not working here for free

I delete it because was totally unprofessional, and is not even related to the PR.

**Comment by BenraouaneSoufiane** [comment]
> > @daimo-eth why you delete my comment?
> > I'm warning you from using my code without paying me (400 USDC), I'm not working here for free
> 
> I delete it because was totally unprofessional, and is not even related to the PR.

As per your reply, you want to copy paste my submission, I'll not stop here

**Comment by Ethanol48** [comment]
> > > @daimo-eth why you delete my comment?
> 
> > > I'm warning you from using my code without paying me (400 USDC), I'm not working here for free
> 
> > 
> 
> > I delete it because was totally unprofessional, and is not even related to the PR.
> 
> 
> 
> As per your reply, you want to copy paste my submission, I'll not stop here

Look, I'm not searching for any problem, they were looking for a spanish native speaker, I contacted and got the assignment.

As per copy and paste, do you know how i18n works? I obviously copy the english file as a template. But it wasn't your code, is the colaborator with the name of `klee` at least they madr the commit for the english file. 

I don't see a problem, let's keep this professional, no need to escalate.

**Comment by BenraouaneSoufiane** [comment]
> > > > @daimo-eth why you delete my comment?
> > 
> > 
> > > > I'm warning you from using my code without paying me (400 USDC), I'm not working here for free
> > 
> > 
> > > 
> > 
> > 
> > > I delete it because was totally unprofessional, and is not even related to the PR.
> > 
> > 
> > As per your reply, you want to copy paste my submission, I'll not stop here
> 
> Look, I'm not searching for any problem, they were looking for a spanish native speaker, I contacted and got the assignment.
> 
> As per copy and paste, do you know how i18n works? I obviously copy the english file as a template. But it wasn't your code, is the colaborator with the name of `klee` at least they madr the commit for the english file.
> 
> I don't see a problem, let's keep this professional, no need to escalate.

They mentioned after I complete 50% of the work, the I've finish it where you don't know even how to start

File: apps/daimo-mobile/src/i18n/languages/es.ts [hunk 1]
```diff
@@ -0,0 +1,794 @@
...
+    },
+  },
+  // -------------------------- SCREENS --------------------------
+  // ------------ HISTORY ------------
+  // HistoryList.tsx
+  historyList: {
+    empty: () => `Ninguna transacción`,
+    screenHeader: {
+      default: () => `Actividad reciente`,
+      other: () => `Between you`, // TODO
```
Comment (kayleegeorge) [inline apps/daimo-mobile/src/i18n/languages/es.ts:33]: note the TODO here

File: apps/daimo-mobile/src/i18n/languages/es.ts [hunk 2]
```diff
@@ -0,0 +1,791 @@
...
+    },
+  },
+  // -------------------------- SCREENS --------------------------
+  // ------------ HISTORY ------------
+  // HistoryList.tsx
+  historyList: {
+    empty: () => `Ninguna transacción`,
+    screenHeader: {
+      default: () => `Actividad reciente`,
+      other: () => `Entre ustedes`,
```
Comment (dcposch) [inline apps/daimo-mobile/src/i18n/languages/es.ts:33]: this was the last one, @Ethanol48 let me know if this looks right!

**Review by dcposch — APPROVED** [review]

File: apps/daimo-mobile/src/i18n/languages/es.ts (hunk 2 above)
Comment (Ethanol48) [reply apps/daimo-mobile/src/i18n/languages/es.ts:33]: Could you provide me with a screenshot when this comes up in the app? To see the context

**Pull request merged**